allure-testng/src/test/resources/suites/parameterized-test.xml

```diff
@@ -2,10 +2,12 @@
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 
 <suite name="Test suite 6">
+    <listeners>
```

Comment:
This seems redundant. The listener is added directly to TestNg https://github.com/allure-framework/allure-java/blob/main/allure-testng/src/test/java/io/qameta/allure/testng/AllureTestNgTest.java#L1277

It's essential to have it like that since we override the writer to have in-memory test results

Reply:
Addressed, thanks @baev.